src/main/app/case/answers/getAnswerRows.test.ts

```diff
@@ -140,9 +140,7 @@ describe('getAnswerRows()', () => {
         applicant1FullNameOnCertificate: 'Sarah Smith',
         applicant2FullNameOnCertificate: 'Billy Bob',
         applicant1LegalProceedings: YesOrNo.YES,
-        applicant1LegalProceedingsRelated: ['marriage', 'property'],
         applicant2LegalProceedings: YesOrNo.YES,
```

Comment:
could you get rid of applicant1LegalProceedings and applicant2LegalProceedings as well?

Reply:
oh right, I didn't think they were